lightning/src/ln/channelmanager.rs

```diff
@@ -2481,18 +2481,18 @@ pub enum RecentPaymentDetails {
 #[derive(Debug, Clone, PartialEq)]
 pub enum Bolt12CreationError {
     /// Error from  BOLT 12 semantic checks.
-    Bolt12SendingError(Bolt12SemanticError),
+    InvalidSemantics(Bolt12SemanticError),
```

Comment:
Commits should be atomic. Let's make a fixup commit for the previous commit (to squash later) for all the renames. `git add -p` is your friend. 🙂